.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
+  "projects": {
+    "default": "insights-frontend-starter-app"
```

Comment:
Are you using Firebase? I think we might be able to remove this file

Reply:
@mturley this file shouldn't be there, it is just the result of copy and paste files. I'll delete it.